Python/Product/Analysis/Interpreter/Ast/AstPythonInterpreterFactory.cs

```diff
@@ -360,7 +364,7 @@ bool useDefaultDatabase
             ImportableModulesChanged?.Invoke(this, EventArgs.Empty);
         }
 
-        protected virtual async Task<IReadOnlyList<string>> GetCurrentSearchPathsAsync() {
+        protected virtual async Task<IReadOnlyList<string>> GetCurrentSearchPathsAsync(CancellationToken cancel) {
```

Comment:
This has only been public API in preview so far, so we can still change it. (Previously the whole class was internal)

Reply:
Should we add `CancellationToken` to other methods of this class as well?